src/index.html

```diff
@@ -15,8 +15,104 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__logo">
+          <a
+            href="#"
+            class="header__logo-link"
+          >
+            <img
+              src="./images/logo.png"
+              alt="logo"
+            />
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
            </li>
            
            <li class="nav__item">
```